### PR TITLE
[HTM-480] Add hidpi settings and services

### PR DESCRIPTION
**Pull request by matthijsln**

### CODE REVIEW

File: docker/db/docker-entrypoint-initdb.d/3-dump.sql.script [hunk 1]
```diff
@@ -48,7 +52,7 @@ ALTER TABLE public.level_ ENABLE TRIGGER ALL;
 ALTER TABLE public.user_ DISABLE TRIGGER ALL;
 
 COPY public.user_ (username, password) FROM stdin;
-admin	{bcrypt}$2a$10$B6My7xzTiTM76Q17.WVvIO8PAsvJuxSGjc0bbUTMCA3kXSuSvycU2
+admin	{bcrypt}$2a$10$yfAqxSZcWXCNSUjTZYvODusTce.B8o8f.pMytcKSPqlJfYwmG0fY6
```
Comment (mprins) [inline docker/db/docker-entrypoint-initdb.d/3-dump.sql.script:55]: this changes the default password no? that means it won't be regenerated in a new deployment

**Review by mprins — DISMISSED** [review]
I don't think the admin password should be changed

File: docker/db/docker-entrypoint-initdb.d/3-dump.sql.script (hunk 1 above)
Comment (mvdstruijk) [reply docker/db/docker-entrypoint-initdb.d/3-dump.sql.script:55]: I tested this config on my local machine and i can login with the password that is generated in the log file. 
So i think this PR is good...

Comment (matthijsln) [reply docker/db/docker-entrypoint-initdb.d/3-dump.sql.script:55]: See the comments in https://github.com/B3Partners/tailormap-viewer/blob/HTM-480-update-config-dump-with-hidpi-settings/docker/db/docker-entrypoint-initdb.d/3-load-data.sh

**Review by mvdstruijk — APPROVED** [review]

**Pull request merged**